tests/test_qpu_braket.py

```diff
@@ -116,6 +116,7 @@ def test_circuit_angle_assignment_qpu_backend_w_hadamard(self):
             qaoa_descriptor, aws_device, shots, None, None, True, 1.0
         )
         qpu_circuit = aws_backend.qaoa_circuit(variate_params)
+        
```

Comment:
are the extra newlines coming out of automated formatting?

Reply:
I'll format all relevant files with black once again before merging.